From d97bab84a0df237d7c6976fa7fc345a0882c2108 Mon Sep 17 00:00:00 2001
From: Aleksey Sanin <aleksey@aleksey.com>
Date: Mon, 24 Oct 2022 22:56:41 +0000
Subject: [PATCH] fix size_max

---
 src/cast_helpers.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/cast_helpers.h b/src/cast_helpers.h
index 67330822..2b670ad1 100644
--- a/src/cast_helpers.h
+++ b/src/cast_helpers.h
@@ -17,13 +17,9 @@
 #endif /* XMLSEC_PRIVATE */
 
 #include <limits.h>
+#include <stdint.h>
 #include "errors_helpers.h"
 
-/* if it is missing */
-#ifndef SIZE_MAX
-#define SIZE_MAX (~(size_t)0)
-#endif /* SIZE_MAX */
-
 /**
  * Helpers for printing out enum values (mostly debugging).
  */
